extensions-contrib/opencensus-extensions/src/main/java/org/apache/druid/data/input/opencensus/protobuf/OpenCensusProtobufExtensionsModule.java

```diff
@@ -45,5 +46,6 @@
   @Override
   public void configure(Binder binder)
   {
+    JsonConfigProvider.bind(binder, "druid.opencensus", OpenCensusProtobufConfig.class);
```

Comment:
I don't think it makes sense for this to be a service-level configuration. Instead, this should be configurable as part off the ingestion spec.

Reply:
@xvrl I looked into the `configuration` of different `extensions` and saw the configuration is always at service-level.

I now also checked `ingestion spec` and it has `data schema`, io config`, `tuning config`. Can you please help me regarding an example extending `ingestion spec` and how the prefix configuration in our case should go and where.

https://druid.apache.org/docs/latest/development/extensions.html
https://druid.apache.org/docs/latest/development/extensions-core/google.html

Reply:
@apoorvmittal10 there are many different types of extensions. Unlike parsers, many extensions cannot be configured at runtime. Parser configurations are defined as part of the constructor, so all you need to make the parser configurable is to add a constructor argument ` @JsonProperty("resourceDimensionPrefix") String resourcePrefix` to `OpenCensusProtobufInputRowParser`

Reply:
Done.